react-frontend/src/components/StyleComponents/headings.js

```diff
@@ -67,9 +67,9 @@ export const SimpleTextPageTitle = styled.h1`
   margin-top: 30px;
 `;
 export const Title2 = styled.h2`
-  font-size: 3rem;
-  font-weight: 700;
-  margin-top: 30px;
+  font-size: 31px;
```

Comment:
This is used on the collaboration tools page.  Are you happy with how that bage looks after these changes?

Reply:
Yes, I checked that out before submitting this PR -- happy with how it looks.